wdl-parsing/WDLParsingFunction/src/main/java/io/dockstore/wdlparser/package-info.java

```diff
@@ -0,0 +1,16 @@
+/*
+ *    Copyright 2020 OICR
```

Comment:
gotta iterate the date and add uc?

Reply:
example?

Reply:
Looks like we've been lazy, but there is https://github.com/dockstore/dockstore-documentation/commit/844ffbb51034d46b423a51bd99e6ed7a179a256e